test/commands/org/authorize.test.ts

```diff
@@ -1,133 +1,123 @@
-import { expect, test } from '@salesforce/command/lib/test';
-import Authorize from '../../../src/commands/hutte/org/authorize';
+import { MockTestOrgData, TestContext } from '@salesforce/core/lib/testSetup';
+import { stubSfCommandUx } from '@salesforce/sf-plugins-core';
+import { stubMethod } from '@salesforce/ts-sinon';
+import { expect } from 'chai';
+import cross_spawn from 'cross-spawn';
+import fs from 'fs';
 import * as api from '../../../src/api';
+import { Authorize } from '../../../src/commands/hutte/org/authorize';
 import * as common from '../../../src/common';
 import * as config from '../../../src/config';
-import * as keychain from '../../../src/keychain';
-import cross_spawn from 'cross-spawn';
 
 describe('hutte:org:authorize', async () => {
-    Authorize.requiresProject = false;
+  const $$ = new TestContext();
```

Comment:
From our previous conversation, I am aware Salesforce follows this naming convention, but I again feel it's not right. I would personally not follow it and ensure a more readable tests codebase.

Reply:
OK 😄